gcp/config.go

```diff
@@ -169,7 +169,7 @@ func (c *Config) SetTags(tags []string) *Config {
 }
 
 func (c *Config) SetImage(version string) *Config {
-	if version == "" {
+	if version == "" || version == "latest" {
```

Comment:
Do we lowercase or otherwise standardise the inputs? Someone will pass `LaTest` I guarantee!